context/llpcCompiler.cpp

```diff
@@ -1943,6 +1943,7 @@ void Compiler::TrimSpirvDebugInfo(
     const uint32_t* pCodePos = pCode + sizeof(SpirvHeader) / sizeof(uint32_t);
 
     uint32_t* pTrimEnd = reinterpret_cast<uint32_t*>(VoidPtrInc(pTrimSpvBin, bufferSize));
+    LLPC_UNUSED(pTrimEnd);
```

Comment:
I think we can remove pTrimEnd if it is not used in any assert.

Reply:
It is used in an assert lower down.